### PR TITLE
Updated Name, Block Number, Moved to Last Call 

### PR DESCRIPTION
**Pull request by MadeofTin**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 2387 state from Draft to Last Call

File: EIPS/eip-2387.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
 eip: 2387
-title: "Hardfork Meta: Mountain Glacier"
+title: "Hardfork Meta: Muir Glacier"
 author: James Hancock (@madeoftin)
 discussions-to: https://ethereum-magicians.org/t/hard-fork-to-address-the-ice-age-eip-2387
 type: Meta
-status: Draft
+status: Last Call
```
Comment (axic) [inline EIPS/eip-2387.md]: Please add `review-period-ends: 12-12-2019` (13th would be two weeks, but that's just Friday 13th :wink:)

**Comment by axic** [comment]
Can you also set EIP-2834 to last call with the same review date in this PR? I'll merge it.

**Comment by axic** [comment]
Ah, also should have updated the title of 2384.

**Pull request merged**